### PR TITLE
Remove ESLint leftovers and refactor TSLint config

### PR DESCRIPTION
**Pull request by k15a**
While working on https://github.com/danger/danger-js/issues/396 I had an issue that I couldn't commit because TSLint was run before Prettier with stylistic rules which are not needed when you also run Prettier. Prettier takes care of this already.

I've also removed the ESLint dead-code because it seems that you've removed ESLint from the project some time ago.

#travial

### CODE REVIEW

File: source/runner/_tests/fixtures/results/__DangerfileBadSyntax.js.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
...
+    "## Error ReferenceError\n```\nhello is not defined\nReferenceError: hello is not defined\n    at Object.<anonymous> (/Users/orta/dev/projects/danger/danger-js/source/runner/_tests/fixtures/__DangerfileBadSyntax.js:4:1)\n    at Module._compile (module.js:624:30)\n    at requireFromString (/Users/orta/dev/projects/danger/danger-js/node_modules/require-from-string/index.js:28:4)\n    at Object.<anonymous> (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:95:21)\n    at step (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:32:23)\n    at Object.next (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:13:53)\n    at /Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:7:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:3:12)\n    at Object.runDangerfileEnvironment (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:66:12)\n```\n### Dangerfile\n```\n1| hello\n--^\n2| \n```\n  "
```
Comment (k15a) [inline source/runner/_tests/fixtures/results/__DangerfileBadSyntax.js.json]: I am not really sure about this one. Tests are passing with and without these changes. As I've removed the `eslint-ignore` from the source file I think the error message should change as well.

**Comment by orta** [comment]
This looks great - in our RN projects we use https://github.com/alexjoverm/tslint-config-prettier to suppress styling rules from tslint - might be easier to do that here too

**Comment by orta** [comment]
Not sure why the build stalled ( I've not seen that before ) but I've re-ran it

File: source/runner/_tests/fixtures/results/__DangerfileBadSyntax.js.json (hunk 1 above)
Comment (orta) [reply source/runner/_tests/fixtures/results/__DangerfileBadSyntax.js.json]: Ah the tests are still in write mode, I should write a danger rule for that

**Pull request merged**